lib/helium/collection.rb

```diff
@@ -115,6 +145,10 @@ def resource_path
       uri.to_s
     end
 
+    def relationship_path
+      URI.parse("#{@belongs_to.resource_path}/relationships/#{@klass.resource_name}").to_s if @belongs_to
```

Comment:
nvm to the nil, I see the add/remove will skip the http method and return itself. do we want to raise if there's no belongs_to, rather than proceeding silently? it seems like something would have to be misconfigured to end up in the place